prime-router/src/main/kotlin/Report.kt

```diff
@@ -1422,11 +1424,18 @@ class Report : Logging {
                     grandParentTrackingValue,
                     null,
                     null,
-                    itemHash
+                    itemHash,
+                    originReportId,
+                    originIndex
                 )
             } else {
                 val trackingElementValue =
                     parentReport.getString(parentRowNum, parentReport.schema.trackingElement ?: "")
+
+                val lineages = WorkflowEngine().fetchItemLineagesForReport(parentReport)
```

Comment:
I'm going to add another tech debt so that we can do this in a more performant manner, but that would be out of scope for this PR